src/components/TimecodeTracker/TimecodeTracker.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import {
+  getTimecodeGameLength,
+  getTimecodeTotalLength,
+} from '../../utils/Timecode'
+import TimecodeInput from './TimecodeInput'
+import TimecodeResult from './TimecodeResult'
+
+export default function TimecodeTracker() {
+  const [timecodes, setTimecodes] = useState(['', '', ''])
+
+  return (
+    <>
+      <form onKeyDown={handleEnter}>
+        {timecodes.map((_, index) => (
+          <TimecodeInput
+            title={index < 2 ? 'Halbzeit ' + (index + 1) : 'O-Töne'}
+            index={index}
+            key={index}
```

Comment:
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
I read about that, but I thought because the list is never reordered or filtered i let the project small, instead of bloated with uuid or shortid. ;-)